examples/go/README.md

```diff
@@ -1,43 +1,47 @@
 ### Go Repo
 
-![](https://github.com/artipie/artipie/workflows/Proof::go/badge.svg)
+[![](https://github.com/artipie/artipie/workflows/Proof::go/badge.svg)](./examples/go)
 
-Try this `go.yaml` file:
+This directory contains a basic example of how artipie can be used as a GOPROXY. 
+Try this example by running `run.sh` script.
+
+Try this `my-go.yaml` file:
 
 ```yaml
 repo:
   type: go
-  path: go
   storage:
     type: fs
-    path: /tmp/artipie/data/go
+    path: /var/artipie/data
 ```
 
-To use it for installing packages add it to `GOPROXY` environment variable:
+After creating the configuration file below, the configured GOPROXY is ready for use. First, you have to declare the following environment variables:
 
 ```bash
-$ export GOPROXY="http://localhost:8080/go,https://proxy.golang.org,direct"
+export GO111MODULE=on
+export GOPROXY=http://localhost:8080/my-go
 ```
 
-Go packages have to be located in the local repository by their
-names and versions, contain Go module and dependencies information
-in `.mod` and `.info` files. Here is an example for package
-`example.com/foo/bar` versions `0.0.1` and `0.0.2`:
+Or, in case you want to stay connected with central proxy.golang.org, declare it in
+the following way:
 
+To use it for installing packages add it to `GOPROXY` environment variable:
+
+```bash
+export GO111MODULE=on
+export GOPROXY=http://localhost:8080/my-go
+export GOPROXY="http://localhost:8080/my-go,https://proxy.golang.org,direct"
```

Comment:
@Sammers21 I do not think we need two configuration examples, `go` developers should know that `GOPROXY` excepts list and it's highly unlikely that someone do not need to stay connected to central